src/mame/includes/raiden2.h

```diff
@@ -3,62 +3,79 @@
 #include "audio/seibu.h"
 #include "machine/seibucop/seibucop.h"
 #include "video/seibu_crtc.h"
+#include <algorithm>
 
-ADDRESS_MAP_EXTERN(raiden2_sound_map, 8);
 ADDRESS_MAP_EXTERN(zeroteam_sound_map, 8);
+GFXDECODE_EXTERN( raiden2 );
+
+#define RAIDEN2_XTAL1   32_MHz_XTAL
+#define RAIDEN2_XTAL2   28.636363_MHz_XTAL
+
+#define PIXEL_CLOCK     RAIDEN2_XTAL2/4
+#define MAIN_CLOCK      RAIDEN2_XTAL1/2
+#define SOUND_CLOCK     RAIDEN2_XTAL2/8
+#define OKI_CLOCK       RAIDEN2_XTAL2/28
+
+ /* hand-tuned to match ~55.47 */
+#define HTOTAL   456
+#define HBEND    0
+#define HBSTART  320
+
+#define VTOTAL   283
+#define VBEND    0
+#define VBSTART  240 // 256 at zeroteam
```

Comment:
Please don't pollute global scope, and especially don't use preprocessor macros for this kind of thing since they produce unhelpful error messages and can't be scoped at all.